tests/test_roundtrip.py

```diff
@@ -73,7 +73,12 @@ def test_multiple_tables(
     if not isinstance(table_name, TableName) or table_name.schema_name is None:
         table_name = TableName("public", table_name)
 
-    assert set(result.keys()) == set((table_name, TableName("public", "table2")))
+    assert set(result.keys()) == set(
+        (
+            ".".join(table_name._unescaped_components),
```

Comment:
Note to future self during review - is this OK to do? I think this will technically break users that want a roundtrip of the TableName, which overloads equality semantics.

How important that is...I don't know.